components/popover/test/utils.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Internal dependencies
+ */
+import { computePopoverPosition } from '../utils';
+
+describe( 'computePopoverPosition', () => {
+	it( 'should leave the position as is there\'s enought space', () => {
+		const anchorRect = {
+			top: 10,
+			left: 10,
+			bottom: 30,
+			right: 30,
+			width: 20,
+			height: 20,
+		};
+
+		const contentSize = {
+			width: 200,
+			height: 300,
+		};
+
+		expect( computePopoverPosition( anchorRect, contentSize, 'bottom right' ) ).toEqual( {
+			contentHeight: null,
+			contentWidth: null,
+			isMobile: false,
+			popoverLeft: 20,
+			popoverTop: 30,
+			xAxis: 'right',
+			yAxis: 'bottom',
+		} );
+	} );
+
+	it( 'should switch to bottom position if there\'s not enough space', () => {
+		const anchorRect = {
+			top: 10,
+			left: 10,
+			bottom: 30,
+			right: 30,
+			width: 20,
+			height: 20,
+		};
+
+		const contentSize = {
+			width: 200,
+			height: 300,
+		};
+
+		expect( computePopoverPosition( anchorRect, contentSize, 'top right' ) ).toEqual( {
+			contentHeight: null,
+			contentWidth: null,
+			isMobile: false,
```

Comment:
All the test cases test the non-mobile case should we add a mobile test case?

Reply:
Yep, good catch it was less important but I'll see if I can tweak the window size.